src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -1547,8 +1547,11 @@ public Iterable<ImportRecordDTO> getImports(
         @Verify(Owner.class) String ownerKey) {
         Owner owner = findOwnerByKey(ownerKey);
 
-        return this.translator.translateQuery(this.importRecordCurator.findRecords(owner),
-            ImportRecordDTO.class);
+        List<ImportRecordDTO> records = new ArrayList<>();
+        this.importRecordCurator.findRecords(owner).stream()
+            .forEach(record -> records.add(this.translator.translate(record, ImportRecordDTO.class)));
```

Comment:
This will do the translator look up for every element in the record. When we're processing streams, use the stream mapper instead:

```
.findRecords(owner)
.stream()
.map(this.translator.getStreamMapper(ImportRecord.class, ImportRecordDTO.class))
```

And then you can just return the mapped stream directly instead of having to recollect it as a list.

Reply:
Good catch! Updated to use the StreamMapper

Reply:
A couple other things that come to mind after having looked at the other CPQ removal PR:

- The title of this commit implies a wider reach than it actually has. We should probably clean that up a bit for the sanity of future maintainers going through the commit history
- CPQ endpoints have implicit paging functionality which will be lost by this change unless the PagingUtil or a custom paging implementation is added.

Reply:
Based on the investigation done when creating this epic, we don't want to do paging for every single endpoint that returns CPQ today. We identified an explicit list of endpoints that either stakeholders (katello/rhsm-api/etc.) already use with paging, or can potentially return a massive amount of results, and we'll only apply paging to those. For this specific PR, the `GET /owners/<orgid>/imports` endpoint is the one that's affected, which isn't in the list of endpoints that we want/need to page

Reply:
Updated the title of the commit!

Reply:
Based on what @nikosmoum said, we need not worry about the loss of paging here; as much as it hurts me to lose.